src/main.py

```diff
@@ -0,0 +1,78 @@
+# sudoku_solver.py
+# author: Daniel Jauergui
+# date: 3-31-2015
+
+from configuration.configuration import *
+from file_manager.file_manager import *
+from menu import Menu
+from game import *
+
+
+def main_menu():
+    menu = Menu('Sudoku Solver')
+    menu.clear_items()
+    menu.add_item((1, 'Configuration Game', config_menu))
+    menu.add_item((2, 'Start Game', game_menu))
+    menu.add_item((0, 'Exit', None))
+    menu.ask()
+
+
+def config_menu():
+    menu = Menu('Sudoku Solver - Config Section')
+    menu.clear_items()
+    menu.add_item((1, 'Select Level', config_menu ))
+    menu.add_item((2, 'Select Algorithm', config_menu ))
+    menu.add_item((3, 'Print Config File', print_config_file ))
+    menu.add_item((4, 'Print Config Level', print_level_file))
+    menu.add_item((5, 'Back', main_menu ))
+    menu.add_item((0, 'Exit', None))
+    menu.ask()
+
+
+def print_config_file():
+    """Print configuration file
```

Comment:
Remember that the documentation of the methods needs to be detailed not mention the same that the method name